pkg/deploy/defaults.go

```diff
@@ -94,6 +95,8 @@ const (
 	DefaultConsoleLinkName                = "che"
 	DefaultConsoleLinkSection             = "Red Hat Applications"
 	DefaultConsoleLinkImage               = "/dashboard/assets/branding/loader.svg"
+	DefaultCheIdentitySecret              = "che-identity-secret"
```

Comment:
Do we want to have a variant based on the flavor ? Something that would be `codeready-identity-secret` in case of CRW ? Not sure, since it seems that the main config map is always called `che`. @nickboldt wdyt ?

Reply:
I can move default value to env variable

Reply:
if @nickboldt thinks it's important why not. As far as I'm concerned, it's OK for my like that.

Reply:
So much in CRW is still called Che. Look at DefaultConsoleLinkName above, for example.

If we can be consistent with naming, then I can programmatically generate CRW operator code from upstream. But I don't care if it's "che-" prefixed or "codeready-" prefixed. 

Both are bad. One exposes project inside product. The other uses a brand as if it's a product name.

Reply:
let's keep it like that for now then.